adops/creative-considerations.md

```diff
@@ -82,12 +82,15 @@ This scenario would require you to create 10,000 line items (10 x 200 x 5). If y
 
 ## Banner and Outstream Creatives
 
-As we mentioned, each creative requires an ad tag script that will call the [Prebid Universal Creative](/overview/prebid-universal-creative.html) to display the correct image.
+As we mentioned, banner creatives require an ad tag script that will render the creative when the bid wins. There are two main options:
 
-One decision you'll want to make is which version of the PUC you want to use. Prebid hosts a copy, which you're free to use. But some publishers might want to use their own copy of the PUC so as to have greater control over the upgrade windows.
+1. Call the [Prebid Universal Creative](/overview/prebid-universal-creative.html) to display the correct image. This also applies to outstream for Prebid.js, though outstream for Mobile requires a video creative.
```

Comment:
Why is the PUC the preferred way of rendering outstream ?

Reply:
Seems like a lot of these problems would go away if the PUC just rendered VAST tags

Reply:
The PUC will never be an outstream renderer. There are dozens (hundreds?) of outstream renderers out there and Prebid gives publishers the choice of which one to use in which scenario.